src/actions/patient.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios';
+import { param } from 'jquery';
+
+
+export function fetchPatients(params= {}) {
+  let queryParams = {_count: 8, _offset:0, '_sort:asc': 'family',...params};
```

Comment:
I meant to write I'd prefer 10. But just to be obstinate I'm going to make it 12.